src/app/shared/top-panel/user-panel/user-menu/user-menu.component.html

```diff
@@ -46,6 +46,15 @@
          aria-hidden="true"></i>&nbsp;
       <span i18n="@@menu.sendFeedback">Send Feedback</span>
     </a>
+    <a (click)="onStartTour()"
+       class="dropdown-item"
+       title="Application tour"
+       i18n-title="@@menu.appTour">
+      <i class="far fa-fw fa-map-signs"
+         data-fa-transform="left-4"
+         aria-hidden="true"></i>&nbsp;
+      <span i18n="@@menu.sendFeedback">Application Tour</span>
```

Comment:
tour vs Tour? (line 51)

Reply:
I followed the pattern of Send Feedback. However, the i18n id was wrong ;-)